### ISSUE TITLE
CTRL+Mousewheel should overwrite the zoom function

### ISSUE THREAD

**Issue by daxtep** [Kyusung4698/PoE-Overlay#369]
This issue was posted and closed again 9 days ago. I think the importance of the problem is viewed as too low.
But this is a huge difference to PoE trade macro, which could users lead to go back to trade macro despite PoE-Overlay beeing much better.

I don't want to disable zoom function at all because sometimes I like to zoom (checking new mtx or stuff). 

Please consider working on that issue to make PoE-Overlay the best addon for trading! :)

**Comment by Kagiri12** [comment]
Yeah I would love to have this fixed asleep, happens too frequently.

**Comment by Kyusung4698** [comment]
Hi, thanks for creating an issue. I understand that this is an annoying issue but for now i'm depended on using [iohook](https://github.com/wilix-team/iohook) for capturing the input. This does not support suppressing the input.

**Comment by markuchi** [comment]
I too would like the zoom to not happen.

**Comment by daxtep** [comment]
@Kyusung4698 Thanks for looking into this.
Maybe it would be possible to find a workaround, like only enabling the zoom when a certain key is pressed? So we disable the zoom in the game options and the key press (SHIFT+Mousewheel for example) forces the zoom?

**Comment by durew** [comment]
Would also love a solution to this!

**Comment by Jerome48** [comment]
The solution to this at the moment is just disabling zoom in with mousewheel in the UI page of PoE's settings

**Comment by durew** [comment]
I found that you can run POE TradeMacro alongside this one, and just disable conflicting functions of one or the other, seems to be working so far

**Comment by daxtep** [comment]
@durew You sir, deserve my respect. Thanks!

**Comment by AmnesiaBR** [comment]
Anyone can simply go to inputs and rebind zoom to Alt+Mouseweel or anything else.
